libs/features/dynamic-navigation/status.js

```diff
@@ -0,0 +1,129 @@
+import { createTag, getConfig, getMetadata } from '../../utils/utils.js';
+import { isDynamicNavDisabled } from './dynamic-navigation.js';
+
+export const ACTIVE = 'active';
+export const ENABLED = 'enabled';
+export const INACTIVE = 'inactive';
+export const tooltipInfo = {
+  active: 'Displayed in green, this status appears when a user is on an entry page or a page with the Dynamic Nav enabled, indicating that the nav is fully functioning.',
+  enabled: 'Displayed in yellow, this status indicates that the Dynamic Nav is set to "on," but the user has not yet visited an entry page.',
+  inactive: 'Displayed in red, this status indicates that the Dynamic Nav is either not configured or has been disabled.',
+};
+
+const getCurrentSource = (status, storageSource, authoredSource) => {
+  if (status === 'on') {
+    return storageSource || authoredSource;
+  }
+  return authoredSource;
+};
+
+const getStatus = (status, disabled, storageSource) => {
+  if (status === 'entry') return ACTIVE;
+
+  if (disabled) return INACTIVE;
+
+  if (status === 'on' && storageSource) return ACTIVE;
+
+  if (status === 'on' && !storageSource) return ENABLED;
+
+  return INACTIVE;
+};
+
+const processDisableValues = (arr, elem) => {
```

Comment:
recommend renaming `arr` since the arg is actually a string.  Something like `values` or `valuesStr`